charts/catalog/templates/apiregistration.yaml

```diff
@@ -13,6 +13,12 @@ spec:
   service:
     namespace: {{ .Release.Namespace }}
     name: {{ template "fullname" . }}-apiserver
+  {{- if eq .Values.apiserver.tls.ca "" }}
+  insecureSkipTLSVerify: true
+  {{- else }}
   caBundle: {{ .Values.apiserver.tls.ca }}
-  priority: 100
+  {{- end }}
+  # Extension API Servers should have a priority value higher than the core API groups that have 100
```

Comment:
The priority field actually split to `groupPriorityMinimum` and `versionPriority` in `v1beta1`. See https://github.com/kubernetes/kubernetes/pull/46800

The ordering was also made more comprehensible (bigger numbers have higher priority). I'd recommend something like:
```
groupPriorityMinimum: 1000
versionPriority: 5
```